odoo_backup_sh/controllers/main.py

```diff
@@ -232,18 +232,14 @@ def fetch_dashboard_data(self):
         dashboard_data.update({
             'configs': backup_configs,
             'notifications': request.env['odoo_backup_sh.notification'].search_read(
-                [('is_read', '=', False)], ['id', 'date_create', 'message'])
+                [('is_read', '=', False)], ['id', 'date_create', 'message']),
+            'up_balance_url': '%s/open_iap_recharge?user_key=%s' % (
+                BACKUP_SERVICE_ENDPOINT,
+                self.get_config_values('options', ['odoo_backup_user_key'])['odoo_backup_user_key']
```

Comment:
Probably POST request is prefered here to don't save user key in browser history